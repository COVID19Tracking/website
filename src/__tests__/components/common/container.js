import React from 'react'
import renderer from 'react-test-renderer'
import Container from '../../../components/common/container'

describe('Components : Common: Container', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Container>
          <p>Sample child</p>
        </Container>,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()

    const textHeavyTree = renderer
      .create(
        <Container textHeavy>
          <p>Sample child</p>
        </Container>,
      )
      .toJSON()
    expect(textHeavyTree).toMatchSnapshot()

    const narrowTree = renderer
      .create(
        <Container textHeavy>
          <p>Sample child</p>
        </Container>,
      )
      .toJSON()
    expect(narrowTree).toMatchSnapshot()
  })
})
