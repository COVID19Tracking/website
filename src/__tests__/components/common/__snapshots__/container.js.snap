// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Common: Container renders correctly 1`] = `
<div
  className="container false container"
>
  <div
    className="layout--content-full"
  >
    <div
      className="layout--content-primary"
    >
      <p>
        Sample child
      </p>
    </div>
  </div>
</div>
`;

exports[`Components : Common: Container renders correctly 2`] = `
<div
  className="container textHeavy container"
>
  <div
    className="layout--content-full"
  >
    <div
      className="layout--content-primary"
    >
      <p>
        Sample child
      </p>
    </div>
  </div>
</div>
`;

exports[`Components : Common: Container renders correctly 3`] = `
<div
  className="container textHeavy container"
>
  <div
    className="layout--content-full"
  >
    <div
      className="layout--content-primary"
    >
      <p>
        Sample child
      </p>
    </div>
  </div>
</div>
`;
